src/io/std_file_descriptor.cr

```diff
@@ -0,0 +1,80 @@
+require "../signal"
+require "c/sys/time"
+require "c/time"
+
+class IO::StdFileDescriptor < IO::FileDescriptor
+  @blocking = true
```

Comment:
```cr
# Never set `O_NONBLOCK` on standard file descriptors! See:
# - https://github.com/crystal-lang/crystal/issues/3674
# - http://cr.yp.to/unix/nonblock.html
property? blocking = true
```
?

Reply:
Doesn't `property?` implies a nilable?

Reply:
@ysbaddaden nope.

Reply:
I'd call `property?` more like query property since it doesn't necessarily have to be boolean.